model/trigger/task.go

```diff
@@ -440,15 +438,11 @@ func (t *taskTriggers) taskRuntimeChange(sub *event.Subscription) (*notification
 	}
```

Comment:
Should we also extract the logic in taskExceedsDuration and similar triggers as well?

Reply:
I originally planned to do that as well, but there are enough small differences that extracting this doesn't make it any more succinct